README.md

```diff
@@ -143,6 +141,9 @@ correct zone, while the error feedback is shown when dropping the item on an
 incorrect drop zone.  In assessment mode, the success and error feedback texts
 are not used.
 
+"Maximum items per Zone" setting controls how much items can be dropped into a 
```

Comment:
@haikuginger "how many items"

Also, it would make sense to move this paragraph to the end of this section: The paragraphs above and below talk about per-item settings, while this one describes a setting that applies to all items.